UEBPrep/app/models/user.rb

```diff
@@ -62,6 +62,7 @@ def self.find_for_twitter_oauth(auth, signed_in_resource=nil)
 
         user = User.create(provider:auth.provider,
                            uid:auth.uid,
+                           username:auth.uid+"@twitter.com",
                            email:auth.uid+"@twitter.com",
```

Comment:
Yeah the twitter login works, but I'll have to see what kind of names twitter passes. This probably shouldn't have been in this build since it was just a test and not an implementation, but it works by auto-generating a username for them. If we can grab an actual name from twitter, that would be better.